RELEASENOTES.md

```diff
@@ -1,5 +1,8 @@
 # Release Notes
 
+### 3.8.2 (UNRELEASED)
```

Comment:
will release right away, bump the date

Reply:
bumped

Reply:
bump the version in functions.php as well 🐈

Reply:
good idear